pkg/cli/admin/inspect/admission_webhooks.go

```diff
@@ -20,41 +21,41 @@ func (c *mutatingWebhookConfigList) addItem(obj interface{}) error {
 	return nil
 }
 
-func gatherMutatingAdmissionWebhook(context *resourceContext, info *resource.Info, o *InspectOptions) error {
+func gatherMutatingAdmissionWebhook(ctx context.Context, rCtx *resourceContext, info *resource.Info, o *InspectOptions) error {
```

Comment:
definitely nit: `rCtx` slightly hurts the readability and I would prefer `resourceContext`.

Reply:
Amended.

Reply:
OMG not really, did case-insensitive replace, ouch 😅